docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
 Eitan Suez <esuez@pivotal.io>
 v0.1, 2017
 :project_id: {{project_id}}
 :domain_name: {{domain_name}}
 :env_name: {{env_name}}
 :director_ip_address: {{director_ip_address}}
+:region: {{region}}
 
+Estimated time:  30 minutes
+
+= Requirements
+
+Lab Requirements
```

Comment:
These are important to keep so that each module can stand on it's own.  So when we create a new course from several modules, the requirements for each module are clearly indicated.  (I didn't have the link to the Lab Requirements so I just pasted the text there as a placeholder.)